app/views/articles/index.html.erb

```diff
@@ -11,11 +11,9 @@
      <link href="https://fonts.googleapis.com/css?family=Varela+Round" rel="stylesheet">
   </head>
```

Comment:
Why are be using a separate head from the one specified in application.html.erb?